### ISSUE TITLE
EUR  not found. Link: xxx

### ISSUE THREAD

**Issue by pReya** [pwaldhauer/amazon-account-crawler#10]
For most orders and years i get a lot of errors like this one:
EUR  not found. Link: /gp/digital/your-account/order-summary.html/ref=oh_aui_dpi_o00_?ie=UTF8&orderID=XXX&print=1

Any idea why that is?

**Comment by pwaldhauer** [comment]
This is caused by digital orders (ebooks, mp3s). The links are different and the script currently is not able to parse them. Someone needs to add support for those.

**Comment by pReya** [comment]
Thanks for the quick answer. Seems like a lot of digital orders are in my account. Don't recall it to be that many, but i'll check again. I'd love to help, but i don't know any ruby ;)

**Comment by pwaldhauer** [comment]
Yes, my account has a lot, too. But since it's always just a small sum (I only buy 3-5€ ebooks) my need to fix this very old code was not that big ;)

Maybe I'll improve this script until I do my next yearly graph... :D
